README.md

```diff
@@ -31,3 +31,9 @@ A few years after that [Lukas Klingsbo](https://github.com/spydon) refactored th
 After this refactor we renamed it to Forge2D since the upstream wasn't maintained to take in our PRs.
 
 There has also been countless other contributors which we are very thankful to!
+
+## Credits
+
+ * The [Flame engine team](https://github.com/orgs/flame-engine/people) who is continuously working on maintaining and improving Flame.
```

Comment:
Improving Forge2D?

Reply:
Yeah, I think you are right, since Forge2D is Flame agnostic

Reply:
Done!